src/main/java/com/codeborne/selenide/appium/SelenideAppium.java

```diff
@@ -0,0 +1,60 @@
+package com.codeborne.selenide.appium;
+
+import com.codeborne.selenide.Selenide;
+import com.codeborne.selenide.WebDriverRunner;
+import javax.annotation.Nonnull;
+
+/**
+ * The main starting point of Selenide-Appium.
+ * <p>
+ * You start with methods {@link #launchApp()} for launching the tested application
+ */
+public class SelenideAppium {
+  private static final AppiumNavigator appiumNavigator = new AppiumNavigator();
+  private static final DeepLinkLauncher deepLinkLauncher = new DeepLinkLauncher();
+
+  /**
+   * The main starting point in your tests.
+   * Launch a mobile application. Do nothing if driver already created.
+   */
+  public static void launchApp() {
+    appiumNavigator.launchApp(() -> WebDriverRunner::getAndCheckWebDriver);
+  }
+
+  /**
+   * Open a deep link for an IOS application
+   * @param deepLinkUrl - deep link url
+   */
+  public static void openIOSDeepLink(@Nonnull String deepLinkUrl) {
+    if (!WebDriverRunner.hasWebDriverStarted()) {
+      launchApp();
+    }
+    deepLinkLauncher.openDeepLinkOnIos(AppiumDriverRunner.getMobileDriver(), deepLinkUrl);
+  }
+  /**
+   * Open a deep link for an Android application
+   * @param deepLinkUrl - deep link url
+   * @param appPackage - Android application package
+   */
+  public static void openAndroidDeepLink(@Nonnull String deepLinkUrl, @Nonnull String appPackage) {
+    if (!WebDriverRunner.hasWebDriverStarted()) {
+      launchApp();
+    }
+    deepLinkLauncher.openDeepLinkOnAndroid(AppiumDriverRunner.getMobileDriver(), deepLinkUrl, appPackage);
+  }
+
+  /**
+   * Terminate application
+   * @param appId - applicationId for Android or bundleId for iOS
+   */
+  public static void terminateApp(String appId) {
+    appiumNavigator.terminateApp(AppiumDriverRunner.getMobileDriver(), appId);
+  }
+
+  /**
+   * Navigate app back to previous screen
+   */
+  public static void back() {
```

Comment:
@BorisOsipov - I think, we have to tweak it a bit.

if( androidDriver()) we can use use back
if(iosdriver() we have to throw exception as back is not supported in ios

Reply:
>back is not supported in ios

It is supported but only for web context or screens with native NavBar component.
I think we should not throw error but maybe add info to javadoc.
https://github.com/appium/appium-xcuitest-driver/blob/master/lib/commands/navigation.js#L11-L38

Wdyt?

Reply:
yes @BorisOsipov you are correct. It is supported in web view and hence we cannot throw any exception.